interests.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Maria S. Interests</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+
+<header>
+		<h1>Maria S.</h1>
+		<nav>
+			<h2><a href="index.html">Resume</a> |
+			<a href="interests.html">Interests</a></h2>
+		</nav>
+	</header>
+
+<br>
+<hr>
+
+
+
+<h2>Interests</h2>
+
+<h3>Electronic Music</h3>
+
+<p>Can't imagine my life without electronic music, and like to go see shows on the reg - will even travel for it. My favorite festivals include FreakNight Seattle, EDC Vegas, Coachella, Hard Summer, Holy Ship edm cruise, EZoo, and others; on the to-do list - TomorrowLand Belgium and Ultra Europe. <br>
+Would like to get into mixing and production as my next hobby. </p>
+
+<img src="photos/holyship8.0.jpeg">
+
+<h3>Innovation</h3>
+
+<p>Apart from my job responsibilities evaluating and mentoring startups, I love contributing to internal quarterly innovation and hack weeks on Biz Dev and Strategy. However, after this course, am hoping to finally be able to participate in internal and external hackathons, as well as personal projects, as a technical team member. </p>
+
+<img src="photos/hackweek6.jpeg">
```

Comment:
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load. 🔨